model/src/main/kotlin/ScanResultContainer.kt

```diff
@@ -46,3 +46,15 @@ data class ScanResultContainer(
      */
     override fun compareTo(other: ScanResultContainer) = id.compareTo(other.id)
 }
+
+/**
+ * Conveniently return all detected licenses for the container's package [id], or an empty set if the container is null.
+ */
+fun ScanResultContainer?.getAllDetectedLicenses() =
```

Comment:
The infered return type is `TreeSet<String>`, better set it to `SortedSet<String>` explicitly.